src/Adaptor.js

```diff
@@ -345,6 +345,143 @@ export function upsertMany(table, uuid, records) {
   };
 }
 
+/**
+ * List the columns of a table in a database.
+ * @example
+ * describeTable(
+ * 'table_name'
+ * )
+ * @constructor
+ * @param {string} table - The name of the table to describe
+ * @returns {Operation}
+ */
+export function describeTable(table) {
+  return state => {
+    let { client } = state;
+
+    try {
+      const query = `SELECT column_name, udt_name, is_nullable
+        FROM information_schema.columns
+        WHERE table_name='${table}';`;
+
+      return new Promise((resolve, reject) => {
+        console.log(`Describing table via : ${query}`);
+
+        client.query(query, (err, result) => {
+          if (err) {
+            reject(err);
+            client.end();
+          } else {
+            //console.log(result);
+            resolve(result);
+          }
+        });
+      }).then(data => {
+        return { ...state, table_data: { body: data } };
+      });
+    } catch (e) {
+      console.log(e);
+      client.end();
+    }
+  };
+}
+
+/**
+ * Create a table in database when given a form definition and a table_name.
+ * @example
+ * insertTable(
+ *  state => state.data.koboColumns, 'table_name'
```

Comment:
Most other functions in here (and across the adaptors) would use `table_name` for the first argument, then "a function returning an array" for the second argument. Is there a specific reason to use table name as arg2? If not, please switch around just so it's more natural WRT the other adaptors.